tsconfig.json

```diff
@@ -31,8 +31,7 @@
       "@katalyst/test-helpers/*": ["content/test/helpers/*"],
       "decentraland-katalyst-contracts/*": ["contracts/*"],
       "decentraland-katalyst-commons/*": ["commons/servers/*"],
-      "decentraland-katalyst-utils/*": ["commons/utils/*"],
-      "katalyst/comms/peer/proto/*": ["dist/bin/comms/peer/proto/*"]
+      "decentraland-catalyst-utils/*": ["commons/utils/*"]
```

Comment:
If we are changing from katalyst to catalyst, shouldn't we do it for all?

Reply:
Yeah... I remember starting that change and reverting it for some reason. Though I cannot remember now why hahaha. I assume it was more difficult that it seemed.